Data/HashMap/Base.hs

```diff
@@ -89,6 +90,12 @@ module Data.HashMap.Base
     , updateOrConcatWithKey
     , filterMapAux
     , equalKeys
+    , lookupRecordCollision
+    , LookupRes(..)
+    , insertNewKey
+    , insertKeyExists
+    , deleteKeyExists
+    , ptrEq
```

Comment:
Why are you adding these exports? I think we should do so in a separate PR if at all.

Reply:
They're needed for the `alterF` version in the `Strict` module that forces the new value before its inserted into the map.

Reply:
Ah, of course. Silly me.